QMMHarmonyShimmer/QMMHarmonyShimmer.cs

```diff
@@ -218,10 +218,21 @@ private static void ShimQMods()
                             }
                         }
                     }
+                    catch (BadImageFormatException)
+                    {
+                        if (Path.GetFileName(filePath).Contains("0Harmony"))
+                            Logger.LogWarning($"QMod in folder {Path.GetDirectoryName(subfolderPath)} is shipping its own version of Harmony! " +
+                                $"This is NOT RECOMMENDED and can lead to serious compatibility issues with other mods! " +
+                                $"If you are a mod author, please do not ship Harmony with your mods, and instead rely on QModManager to load it for you.");
```

Comment:
Add a link to the QModManager wiki in this log where we detail where the new Harmony DLL is located.

Reply:
4734fe3